src/toast/util/toastReducer.ts

```diff
@@ -57,6 +64,29 @@ function toastReducer(state: ToastState, action: ToastAction): ToastState {
       break;
     }
 
+    case "SET_LIMIT": {
+      const {limit} = action;
+      // prune the toasts exceeding the limit
+      const toastStack = state.toastStack.slice(
+        Math.max(0, state.toastStack.length - limit)
+      );
+
+      newState = {
+        ...state,
+        toastStack,
+        limit
+      };
+      break;
+    }
+    case "SET_AUTO_CLOSE": {
+      const {autoCloseToasts} = action;
+
+      newState = {
+        ...state,
+        autoCloseToasts
+      };
+      break;
+    }
     default:
```

Comment:
```suggestion
      break;
    }
    
    case "SET_AUTO_CLOSE": {
      const {autoCloseToasts} = action;

      newState = {
        ...state,
        autoCloseToasts
      };
      break;
    }
    
    default:
```

Let's give them some space